homeassistant/components/scrape/sensor.py

```diff
@@ -247,11 +249,20 @@ def _async_update_from_rest_data(self) -> None:
         value = self._extract_value()
 
         if self._value_template is not None:
-            self._attr_native_value = (
-                self._value_template.async_render_with_possible_json_value(value, None)
+            value = self._value_template.async_render_with_possible_json_value(
+                value, None
             )
```

Comment:
Ifyou use a walrus, it will fit on two lines intead of four:
```python
        if (template := self._value_template) is not None:
            value = template.async_render_with_possible_json_value(value, None)
```